Casks/puppet-agent-5.rb

```diff
@@ -18,8 +18,8 @@
     sha256 'be76193cd6298e372667fc65381a7f711c0585a2f70c355014dc4d59ae8948f6'
   else
     os_ver = '10.14'
-    version '5.5.18'
-    sha256 '06bfdf742356edbd4532246134a3bae43e36faa4daacc94017da760fdb3f9867'
+    version '5.5.19'
+    sha256 '836fc6a5ae0cd0da0de9ac4d66821c4da958a65840ef6cc427a877266a77c566'
```

Comment:
This should also include the package for 10.15. I believe this is done with a modification to the rakefile here: https://github.com/puppetlabs/homebrew-puppet/blob/master/Rakefile#L48

Reply:
good catch